pkg/services/ngalert/state/manager.go

```diff
@@ -39,17 +39,26 @@ type Manager struct {
 	externalURL   *url.URL
 }
 
-func NewManager(metrics *metrics.State, externalURL *url.URL, instanceStore InstanceStore, images ImageCapturer, clock clock.Clock, historian Historian) *Manager {
+type ManagerCfg struct {
```

Comment:
There's only one thing to configure here - would have love for this to be simply `Configuration` as it is pretty well isolated as part of the `state` package.